config/locales/en.yml

```
@@ -380,6 +380,9 @@ en:
         filter_case_type: Type
         filter_open_case_status: Status
         filter_sensitivity: Sensitivity
+        filter_complaint_type: Complaint type
+        filter_complaint_priority: Sensitivity
```

I think we've got into a bit of a muddle in the ticket regarding sensitivity and priority. These are two separate things and arguably each requires its own filter options. Sensitivity refers to high profile cases (in the news, etc) whereas priority is a separate field that we introduced for complaints
```suggestion
        filter_complaint_priority: Priority
```